frontend/client/components/CreateFlow/index.tsx

```diff
@@ -337,7 +338,15 @@ class CreateFlow extends React.Component<Props, State> {
     }
   };
 
-  private openPublishWarning = () => {
+  private openPublishWarning = async () => {
+    if (this.props.form) {
+      try {
+        const { data: invites } = await getProposalInvites(this.props.form.proposalId);
```

Comment:
Just wanted to point out that this proposal invite check needs to complete before the modal will open. 

If there's a noticeable lag in staging (or if we're worried about UX under poor connectivity), maybe we should add a spinner somewhere.